src/convert.jl

```diff
@@ -130,6 +130,32 @@ function convert(::Type{VPolygon}, P::AbstractPolytope)
     return VPolygon(vertices_list(P))
 end
 
+"""
+    convert(::Type{VPolygon}, X::LazySet)
+
+Generic conversion to polygon in vertex representation.
+
+### Input
+
+- `type` -- target type
+- `X`    -- set
+
+### Output
+
+The 2D set represented as a polygon.
+
+### Algorithm
+
+First we concretize `X` obtaining a (possibly) new representation `Xc`; then we
+wrap the result into a polygon in vertex representation by computing the list of
+vertices of `Xc`. The vertices of the resulting polygon in vertex representation
+are guaranteed to be sorted in counter-clockwise fashion.
+"""
+function convert(::Type{VPolygon}, X::LazySet)
+    @assert dim(X) == 2 "set must be two-dimensional for conversion"
+    return VPolygon(vertices_list(concretize(X)))
```

Comment:
@schillic what do you think about using `concretize` here?

Reply:
i would prefer to not use it, but note that, for example,

```julia
julia> vertices_list(rand(2, 10) * rand(Zonotope, dim=10))
ERROR: AssertionError: package 'Polyhedra' not loaded (it is required for executing `remove_redundant_vertices`)
Stacktrace:
 [1] macro expansion at /home/mforets/.julia/dev/LazySets/src/Assertions/Assertions.jl:23 [inlined]
....

julia> vertices_list(concretize(rand(2, 10) * rand(Zonotope, dim=10)))
36-element Array{Array{Float64,1},1}:
 [-31.25354193606883, -22.148084939907815]
 [-30.664022201688585, -23.27061463165326]
...
```

Reply:
Agreed, I would delegate this decision to `vertices_list` instead. We could let `vertices_list` use `concretize`, possibly only if `Polyhedra` is not loaded.